packages/dd-trace/src/telemetry/log_collector.js

```diff
@@ -0,0 +1,133 @@
+'use strict'
+
+const { calculateDDBasePath } = require('../util')
+const log = require('../log')
+
+const ddBasePath = calculateDDBasePath(__dirname)
+const EOL = '\n'
+const STACK_FRAME_LINE_REGEX = /^\s*at/gm
```

Comment:
If a line start just with "at", like "atomic" it will match.
With the current regex:
https://regex101.com/r/70BssI/1

I propose you this other regex, just adding another `\s` after `at`
Proposed regex:
https://regex101.com/r/6gKJDz/1

Reply:
thkx